package.json

```diff
@@ -41,6 +41,7 @@
     "bootstrap": "^4.6.0",
     "c0d3-diff": "^0.0.1",
     "dayjs": "^1.10.5",
+    "discord.js": "^12.5.3",
```

Comment:
I might have missed it but if this is just added for type defs can it be moved to dev dependency ? or does that break our ci?

Reply:
Should work in devDependencies too, it was @songz 's suggestion that main dependencies is a better place to put it in to prevent future issues.